backend/serialization/vanilla_LibBackend-Pusher_LibBackend-Pusher-AddOpEventTooBigPayload_simple.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
Unclear why this is gone.

Reply:
We don't yet handle the case of the Pusher.com AddOpV1 payload being too big to push to the client.

```
let tooBigPayload = { tlids = tlids } |> Json.Vanilla.serialize
// CLEANUP: when changes are too big, notify the client to reload them. We'll
// have to add support to the client before enabling this. The client would
// reload after this.
// push canvasID "addOpTooBig" tooBigPayload
```

is the code before changes, and we have the equivalent (just swallow the issue) in the new code.

Since we don't yet serialize it, I figured we didn't need to register the serializer, nor test for how it serializes.